common/source/docs/common-scripting-parameters.rst

```diff
@@ -0,0 +1,80 @@
+.. common-scripting-parameters:
+
+=======================================
+Accessing/Adding Parameters via Scripts
+=======================================
+
+LUA scripts have the ability to access and change ArduPilot parameters setup by the firmware.
+
+for example: ::
+
+    local parameter = parm:get('SCR_ENABLE')
+
+will retrieve the SCR_ENABLE parameter and assign it to the variable "parameter"
+
+However, looking up parameters with a name search, as the above code does, is slow.  You can create a local variable using the Parameter user data method which avoids a name search every use and is much faster: ::
+
+   local SCR_ENABLE = Parameter()      --creates a parameter object
+   SCR_ENABLE.init('SCR_ENABLE')       --get the physical location in parameters memory area so no future name search is done
+   local parameter = SCR_ENABLE.get()  --retrieve that parameters value and assign to "parameter"
+
+the current parameter access/modify methods, either by name or as a Parameter object method are:
+
+- init(name)                          -initializes a parameter object
+- configured()                        -check to see if a Parameter object has been initialized
+- get(name or void)                   -retrieves the value
```

Comment:
```suggestion
- get()                   -retrieves the value
```
On parameter object you can only get with no name. Get with a name is for `param:` global only. 

Like wise on the rest.

Reply:
that was what I was trying to convey.....get() for param object get(name) for param:...suggestions for a way to show that the argument is name for one and nothing(void) for the other?